packages/protocol/contracts/common/Accounts.sol

```diff
@@ -85,102 +133,191 @@ contract Accounts is IAccounts, ReentrancyGuard, Initializable, UsingRegistry {
 
   /**
    * @notice Authorizes an address to sign votes on behalf of the account.
-   * @param voter The address of the vote signing key to authorize.
+   * @param signer The address of the signing key to authorize.
    * @param v The recovery id of the incoming ECDSA signature.
    * @param r Output value r of the ECDSA signature.
    * @param s Output value s of the ECDSA signature.
-   * @dev v, r, s constitute `voter`'s signature on `msg.sender`.
+   * @dev v, r, s constitute `signer`'s signature on `msg.sender`.
    */
-  function authorizeVoteSigner(address voter, uint8 v, bytes32 r, bytes32 s) external nonReentrant {
+  function authorizeVoteSigner(address signer, uint8 v, bytes32 r, bytes32 s)
```

Comment:
Is this where the BLS rotation will be implemented?

Reply:
Nope! This is actually unrelated, and allows you to specify a different ECDSA key for voting in governance in validator elections.

BLS rotation is implemented in `Validators.updateBlsKey`

Reply:
OK! I understand it better now.